website/addons/dropbox/utils.py

```diff
@@ -79,7 +79,12 @@ def log(self, action, extra=None, save=False):
 
 
 def is_subdir(path, directory):
-    if not (path and directory):
+    if not path:
+        return False
+    # directory is root directory
+    if directory == '' or directory == '/':
```

Comment:
@sloria Currently there is a <code>clean_path</code> function in this file which removes the beginning and trailing '/'s, A lot of times this function is called in this way <code>is_cubdir(clean_path(path), clean_path(directory))</code>.